src/components/UI/Modal.css

```diff
@@ -12,6 +12,16 @@
   transition: all 0.3s ease-out;
 }
 
+.ModalBackdrop {
```

Comment:
Is it more common to have individual scoped classes like this rather than reusability? ie: this backdrop was used in the Modal and the SideDrawer but is now broken out into 2 separate CSS classes. I thought `CSSModules` does something similar already?